mscore/editstyle.cpp

```diff
@@ -1111,6 +1111,8 @@ void EditStyle::setValues()
       musicalTextFont->addItem("Emmentaler Text", "MScore Text");
       musicalTextFont->addItem("Gonville Text", "Gootville Text");
       musicalTextFont->addItem("MuseJazz Text", "MuseJazz Text");
+      musicalTextFont->addItem("Petaluma Text", "Petaluma Text");
+      musicalTextFont->addItem("Petaluma Script", "Petaluma Script");
```

Comment:
Not sure we need Petaluma Script here, it doesn't seem to be a musical text font

Reply:
About all the fonts variants (petaluma script, petaluma text), I would keep all of these since I do expect the fonts to be consistent throughout my parts, so I often find myself changing _all_ text styles to musejazz when arranging for jazz. 
I would not be pleased to be forced not to use Petaluma just because it doesn't fit with the rest of the text.

Reply:
Hmmm. I misread your comment. You are right. I don't think Petaluma Script belongs here.